src/client.h

```diff
@@ -417,6 +417,7 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 	bool checkPrivilege(const std::string &priv) const
 	{ return (m_privileges.count(priv) != 0); }
 
+	const std::unordered_set<std::string> &getPrivilegeList() {return m_privileges;};
```

Comment:
the extra ; after braces is not needed, i you can add space inside braces too can be nice :). Also this function should be const (sorry for to be late about that)